### PR TITLE
Use the current selected user time format

### PR DESCRIPTION
**Pull request by tintou**
We then need to replace the GSettings with this DBus parameter that is now exposed

### CODE REVIEW

**Comment by peteruithoven** [comment]
This is great! This should fix quite some issues. 
What's the relationship between the `io.elementary.greeter.AccountsService` and the `org.freedesktop.Accounts`? 
This accounts service could it also store & share things like the display, power, etc settings?

**Comment by peteruithoven** [comment]
This should fix: #275

File: data/io.elementary.greeter.AccountsService.policy [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                <message gettext-domain="io.elementary.greeter">Authentication is required to set one's own greeter properties.</message>
+                <defaults>
+                        <allow_any>yes</allow_any>
+                        <allow_inactive>yes</allow_inactive>
+                        <allow_active>yes</allow_active>
+                </defaults>
+        </action>
+
+        <action id="io.elementary.greeter.AccountsService.ReadAny">
+                <description gettext-domain="io.elementary.greeter">Set properties of any user</description>
```
Comment (vjr) [inline data/io.elementary.greeter.AccountsService.policy]: This should say `Get properties...` instead of `Set properties...` ?

**Comment by tintou** [comment]
> This is great! This should fix quite some issues.
> What's the relationship between the `io.elementary.greeter.AccountsService` and the `org.freedesktop.Accounts`?
> This accounts service could it also store & share things like the display, power, etc settings?
> 
> I found some info on this AccountsService: https://www.freedesktop.org/wiki/Software/AccountsService/
> It's a pity they've not made the actual docs easily accessible.

AccountsService has now an extension system, it is not really well documented but there is at least one example here https://gitlab.freedesktop.org/accountsservice/accountsservice/blob/master/doc/org.mydm.MyDisplayManager.Wallpaper.xml
The DBus interface is then "mounted" by the accountsservice daemon which is available to the whole system.

File: data/io.elementary.greeter.AccountsService.policy (hunk 1 above)
Comment (tintou) [reply data/io.elementary.greeter.AccountsService.policy]: Thanks for noticing, it's now fixed

**Comment by decathorpe** [comment]
So, are these new datetime settings only going to work if the session was started from the elementary greeter? I assume GDM et al. don't support this accountsservice extension (yet).

**Comment by tintou** [comment]
As long as the AccountsService files are installed it should work. GDM won't use it of course as they are not interested in using the user settings there

**Comment by decathorpe** [comment]
That's great news :) Thanks for the clarification.

**Comment by getsnoopy** [comment]
When can this change be expected to go into a release?

**Comment by cassidyjames** [comment]
It's on the todo list for the next release, so any day now. :smile:

**Pull request merged**